src/AutomatedCar/SystemComponents/CarCollisionDetector.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace AutomatedCar.SystemComponents
+{
+    using AutomatedCar.Models;
+    using AutomatedCar.SystemComponents.Packets;
+    using Avalonia;
+    using Avalonia.Media;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    public class CarCollisionDetector : SystemComponent
+    {
+        private CollisionPacket packet;
+        private IEnumerable<WorldObject> collidableWorldObjects;
+
+        public CarCollisionDetector(VirtualFunctionBus virtualFunctionBus)
+            : base(virtualFunctionBus)
+        {
+            this.packet = new CollisionPacket();
+            this.collidableWorldObjects = World.Instance.WorldObjects
+                                        .Where(obj => obj.Collideable
+                                        && obj != World.Instance.ControlledCar);
```

Comment:
`&&` should be at the end of the `obj.Collideable` of you can break `&&` to a whole new line making this 3 lines altogether.